valuesfile/valuesfile.go

```diff
@@ -15,7 +15,7 @@ type renderer struct {
 func NewRenderer(readFile func(filename string) ([]byte, error), basePath string, env environment.Environment) *renderer {
 	return &renderer{
 		readFile:         readFile,
-		tmplFileRenderer: tmpl.NewFileRenderer(readFile, basePath, env),
+		tmplFileRenderer: tmpl.NewFileRenderer(readFile, basePath, env, ""),
```

Comment:
We should probably make `{{ .Namespace }}` in values.yaml.gotmpl files fail rather than returning an empty string. But that's an another story! Let me just note about it here as a reminder

Reply:
Okay. Please let me know if you'd like me to make any changes here!